src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -283,6 +283,8 @@ const ViewPopup = (() => {
             $container.find('#notice_ongoing').setVisibility(1);
         }
 
+        Clock.setExternalTimer(is_ended ? '' : updateTimers); // stop/resume timer
```

Comment:
we need to restart the timer when visiting portfolio after visiting profit table or statement page